test/simpleTestForGenerator/CC-PDM-1.0.txt

```diff
@@ -0,0 +1,27 @@
+No Copyright
+
+This work has been identified as being free of known restrictions under
+copyright law, including all related and neighboring rights.
+
+
+You can copy, modify, distribute and perform the work, even for commercial
+purposes, all without asking permission. See Other Information below.
+
+Other Information
+
+The work may not be free of known copyright restrictions in all jurisdictions .

```

Comment:
Note the stray space here before the period. 

The CC site appears to have the stray space. [Scancode](https://github.com/aboutcode-org/scancode-toolkit/blob/develop/src/licensedcode/data/licenses/cc-pdm-1.0.LICENSE) doesn't. Some of the linked examples have it, others don't.

For SPDX matching purposes, it shouldn't matter, because whitespace will be ignored under the SPDX matching guidelines. So I'm inclined to leave this as-is, even though there's a difference here between the test text version and the XML file.